pallets/attesters/src/lib.rs

```diff
@@ -789,32 +1489,59 @@ pub mod attesters_test {
         });
     }
 
-    fn sign_and_submit_attestation(
+    #[test]
+    fn deregister_attester_releases_all_funds() {
+        let mut ext = ExtBuilder::default().build();
+        ext.execute_with(|| {
+            let _attester_info = register_attester_with_single_private_key([1u8; 32]);
+            let attester_account_id = AccountId::from([1u8; 32]);
+
+            deregister_attester(attester_account_id);
+        });
```

Comment:
There's no assertion here, i'd also like to see a path where we force any of those panics inside

Reply:
assertions covered in deregister_attester, will add unhappy paths